help/guides/pivot-table.md

```diff
@@ -0,0 +1,93 @@
+
+Similarly, in Freetext.ai, you can build pivot tables to summarize customer feedback. Start by selecting the data you want to analyze, then choose how to categorize it with Row and Column Dimensions and what information to highlight (metrics). For detailed instructions on how to add a pivot table to a new report, visit the [Create Widget](/reports/widgets.md) section of our documentation.
+
+### Dimensions
+
+Dimensions are characteristics that add context to your metrics. They help categorize and group data for analysis. Dimensions can be time, geographical location, product categories, customer segments, or marketing channels. Select dimensions to break down your metrics into categories.
+
+Row and column dimensions categorize and organize data hierarchically, based on selected categories, with the option to select up to four categories for each dimension. The row dimensions create groupings or partitions of the data, while column dimensions create sub groupings within those row groups. 
+
+### Metrics
+
+Metrics are numerical measurements that represent different aspects of business. They provide specific data points like Records Count, product ratings, customer satisfaction score, Impact Score, etc. Each cell presents the metric value corresponding to the specific combination of categories defined by the row and column dimensions. You can select up to 3 metrics.
+
+![create-pivot](/img/help/guides/pivot-table/create-pivot-table.png)
+
+### Filters
+
+You can use filters to focus on specific subsets of data within your pivot table. Eg. If Time is selected as either the row or column dimension, you can use the Time and Graph Filters to refine the data further.
+
+### Example
+
+To illustrate the usage of pivot tables in Freetext.ai, let's consider a scenario where we want to analyze the average review rating and review volume for different products of a brand across various sentiment polarities (positive, negative, or neutral).
+
+Row Dimensions:
+1. Entity
+2. Sentiment
+   
+Column Dimensions:
+1. Time
+   
+Metrics:
+1. Records(Count)
+2. Review Rating Value(Avg)
+
+With these dimensions and metrics, the pivot table would look like this:
+
+![pivot-example-freetext](/img/help/guides/pivot-table/pivot-example-freetext.png)
+
+The intention behind creating this pivot table is to gain insights into the following:
+
+1. Identify the products that are performing well or poorly in terms of customer satisfaction, as indicated by the average review rating value for positive, negative, and neutral sentiments.
+2. Understand the distribution of review volume (record count) across different sentiment polarities for each product.
+3. Analyze trends over time by looking at the year-over-year changes in average review ratings and review volume for each product and sentiment polarity.
+
+### Sorting
+
+When the table has loaded, you can choose to sort it by a single column by clicking on the row/column name.
+
+### Design
```

Comment:
(Later) might want to pull out design as a separate doc "Design Widget" because we'll be adding many settings.

> Overall settings for widget (card/header/etc)
> Settings for Table/Pivot Table
> Settings for Graphs